<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 4 Application | Geometry and topology statistical inference with applications.</title>
  <meta name="description" content="Capstone Project." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 4 Application | Geometry and topology statistical inference with applications." />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Capstone Project." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 Application | Geometry and topology statistical inference with applications." />
  
  <meta name="twitter:description" content="Capstone Project." />
  

<meta name="author" content="Emil Le" />


<meta name="date" content="2021-03-05" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="persistence-homology.html"/>
<link rel="next" href="random.html"/>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-3.5.1/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="libs/rglWebGL-binding-0.104.16/rglWebGL.js"></script>
<link href="libs/rglwidgetClass-0.104.16/rgl.css" rel="stylesheet" />
<script src="libs/rglwidgetClass-0.104.16/rglClass.min.js"></script>
<script src="libs/CanvasMatrix4-0.104.16/CanvasMatrix.min.js"></script>
<link href="libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<link href="libs/combineWidgetStyle-0.1/combineWidgets.css" rel="stylesheet" />
<script src="libs/combineWidgets-binding-0.10.1/combineWidgets.js"></script>
<script src="libs/rglPlayer-binding-0.104.16/rglPlayer.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#homeomorphism"><i class="fa fa-check"></i><b>1.1</b> Homeomorphism</a></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#differential-manifold"><i class="fa fa-check"></i><b>1.2</b> Differential Manifold</a></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#probability-measure"><i class="fa fa-check"></i><b>1.3</b> Probability measure</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html"><i class="fa fa-check"></i><b>2</b> Statistical shape matching</a>
<ul>
<li class="chapter" data-level="2.1" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html#intrinsic-data-analysis"><i class="fa fa-check"></i><b>2.1</b> Intrinsic data analysis</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html#fréchet-mean-and-variance"><i class="fa fa-check"></i><b>2.1.1</b> Fréchet mean and variance</a></li>
<li class="chapter" data-level="2.1.2" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html#geodesics-and-arc-distance"><i class="fa fa-check"></i><b>2.1.2</b> Geodesics and arc distance</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html#central-limit-theorem"><i class="fa fa-check"></i><b>2.2</b> Central Limit Theorem</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html#sample-estimator-for-frechet-mean"><i class="fa fa-check"></i><b>2.2.1</b> Sample Estimator for Frechet mean:</a></li>
<li class="chapter" data-level="2.2.2" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html#sample-estimator-for-frechet-variance"><i class="fa fa-check"></i><b>2.2.2</b> Sample Estimator for Frechet variance:</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html#geodesics-regression"><i class="fa fa-check"></i><b>2.3</b> Geodesics Regression</a></li>
<li class="chapter" data-level="2.4" data-path="statistical-shape-matching.html"><a href="statistical-shape-matching.html#isomorphic-embedding"><i class="fa fa-check"></i><b>2.4</b> Isomorphic embedding</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="persistence-homology.html"><a href="persistence-homology.html"><i class="fa fa-check"></i><b>3</b> Persistence Homology</a>
<ul>
<li class="chapter" data-level="3.1" data-path="persistence-homology.html"><a href="persistence-homology.html#vietorisrips-complex"><i class="fa fa-check"></i><b>3.1</b> Vietoris–Rips complex</a></li>
<li class="chapter" data-level="3.2" data-path="persistence-homology.html"><a href="persistence-homology.html#filtrations"><i class="fa fa-check"></i><b>3.2</b> Filtrations</a></li>
<li class="chapter" data-level="3.3" data-path="persistence-homology.html"><a href="persistence-homology.html#reeb-graph-inference"><i class="fa fa-check"></i><b>3.3</b> Reeb Graph inference</a></li>
<li class="chapter" data-level="3.4" data-path="persistence-homology.html"><a href="persistence-homology.html#confidence-set"><i class="fa fa-check"></i><b>3.4</b> Confidence Set</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="application.html"><a href="application.html"><i class="fa fa-check"></i><b>4</b> Application</a>
<ul>
<li class="chapter" data-level="4.1" data-path="application.html"><a href="application.html#gene-expression"><i class="fa fa-check"></i><b>4.1</b> Gene Expression</a></li>
<li class="chapter" data-level="4.2" data-path="application.html"><a href="application.html#generative-model-for-gene-co-expression-network"><i class="fa fa-check"></i><b>4.2</b> Generative model for gene co-expression network</a></li>
<li class="chapter" data-level="4.3" data-path="application.html"><a href="application.html#data-collection-and-pre-processing"><i class="fa fa-check"></i><b>4.3</b> Data Collection and Pre-processing</a></li>
<li class="chapter" data-level="4.4" data-path="application.html"><a href="application.html#network-visualization"><i class="fa fa-check"></i><b>4.4</b> Network Visualization</a></li>
<li class="chapter" data-level="4.5" data-path="application.html"><a href="application.html#heavy-tailed-distribution-analysis"><i class="fa fa-check"></i><b>4.5</b> Heavy tailed distribution analysis</a></li>
<li class="chapter" data-level="4.6" data-path="application.html"><a href="application.html#betti-number-inference"><i class="fa fa-check"></i><b>4.6</b> Betti Number Inference</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="random.html"><a href="random.html"><i class="fa fa-check"></i><b>5</b> random</a></li>
<li class="chapter" data-level="6" data-path="code-need-to-fix.html"><a href="code-need-to-fix.html"><i class="fa fa-check"></i><b>6</b> code need to fix</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Geometry and topology statistical inference with applications.</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="application" class="section level1" number="4">
<h1><span class="header-section-number">Chapter 4</span> Application</h1>
<div id="gene-expression" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Gene Expression</h2>
<p>The task of classifying the cancer regulatory pathways of patients becomes important, and mathematics have provided us the tools to do so. The gene expression is one of many valuable biometric information to identify cancer cells, in particular, the mRNA copies presented in tumors. From this data, one can construct the gene co-expression network in order to analyze the behavior of active cancer genes. This capstone focuses on analyzing the difference of topology throughout AML networks in order to see if one can predict or dentify the biomark of AML to improve the prognosis of the disease or discover effective drug pathways.</p>
</div>
<div id="generative-model-for-gene-co-expression-network" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Generative model for gene co-expression network</h2>
<p>Define the gene co-expression network that we want as <span class="math inline">\(G = (\mathcal{V}, \mathcal{E})\)</span>, where the vertex set <span class="math inline">\(\mathcal{V}\)</span> consists of genes. An edge in the edge set <span class="math inline">\(\mathcal{E}\)</span> denotes if there is similarity in expression level between 2 genes, which results in G being an undirected network.
For constant threshold <span class="math inline">\(\epsilon = 10^{-6}\)</span>, let the metric space <span class="math inline">\((X,\partial_X)\)</span> where <span class="math inline">\(X\)</span> is the gene expression matrix and <span class="math inline">\(\partial_X: X\times X \to \mathbb{R}\)</span> defined by the Pearson’s correlation:
<span class="math display">\[\partial_X = \frac{\\Cov(X,X)}{\sigma_X^2}\]</span>
Let the filter function <span class="math inline">\(f: X \to \mathbb{R}^{n}\)</span> finds the first 2 nearest neighbors of each element in <span class="math inline">\(X\)</span> in the Euclidean norm <span class="math inline">\(\mathcal{L}^2\)</span>, then <span class="math inline">\(\mathcal{C} = \{U_{\alpha}\}\)</span> is a cover of the range of <span class="math inline">\(f\)</span> in <span class="math inline">\(\mathbb{R}\)</span>. Define the equivalence relation, <span class="math inline">\(\sim\)</span>, such that <span class="math inline">\(a \sim b\)</span> if <span class="math inline">\(a,b \in f^{-1}(U_{\alpha})\)</span> for arbitrary <span class="math inline">\(\alpha\)</span>. The Reeb space of X is the quotient of X by this equivalence relation. For appropriate choices of interval and percentages overlaps, we can have:</p>
<p></p>
<p> Initialized empty graph G = <span class="math inline">\((\mathcal{V},\mathcal{E})\)</span>;</p>
<p> Initialized parameters <span class="math inline">\(\partial_X\)</span>, <span class="math inline">\(f\)</span>, <span class="math inline">\(\mathcal{C}\)</span> as stated;</p>
<p>  discretization of Reeb space of X :</p>
<p> Cluster each inverse image <span class="math inline">\(f^{-1}(U_{\alpha})\subseteq X\)</span> <span class="math inline">\(\forall U_{\alpha} \in \mathcal{C}_{\alpha,i}\)</span>;</p>
<p> Add elements in <span class="math inline">\(\mathcal{C}_{\alpha,i}\)</span> to <span class="math inline">\(\mathcal{V}\)</span>;</p>
<p> If <span class="math inline">\(\mathcal{C}_{\alpha,i} \cap\mathcal{C}_{\alpha^{\prime},i^{\prime}} = \emptyset\)</span> then add <span class="math inline">\(e_{\mathcal{C}_{\alpha,i},\mathcal{C}_{\alpha^{\prime},i^{\prime}}}\)</span> to <span class="math inline">\(\mathcal{E}\)</span>;</p>
<p> Color <span class="math inline">\(v \in \mathcal{C}_{\alpha,i} \subset \mathcal{V}\)</span> by average value of <span class="math inline">\(f\)</span> on <span class="math inline">\(v\)</span>.</p>
<p></p>
<p> G = <span class="math inline">\((\mathcal{V},\mathcal{E})\)</span></p>
<p></p>
</div>
<div id="data-collection-and-pre-processing" class="section level2" number="4.3">
<h2><span class="header-section-number">4.3</span> Data Collection and Pre-processing</h2>
<p>I used the gene expression dataset from The Cancer Genome Atlas (TCGA) for both the training set (TCGA-LAML) and the validation set (query sample from healthy bone marrow and blood tissues). The data is obtained experimentally through Illumina HiSeq 2000 RNA Sequencing, which are <span class="math inline">\(log_2(x+1)\)</span>-RSME normalized with 95% credibility interval.</p>
<p><img src="/Users/macalesteritsloaner/Downloads/bookdown-demo-master/dataset/sampleclustering.png" /></p>
<p>Heat map:
<img src="/Users/macalesteritsloaner/Downloads/bookdown-demo-master/dataset/heatmap.png" /></p>
</div>
<div id="network-visualization" class="section level2" number="4.4">
<h2><span class="header-section-number">4.4</span> Network Visualization</h2>
<p><img src="/Users/macalesteritsloaner/Downloads/bookdown-demo-master/dataset/genenetwork.png" /></p>
</div>
<div id="heavy-tailed-distribution-analysis" class="section level2" number="4.5">
<h2><span class="header-section-number">4.5</span> Heavy tailed distribution analysis</h2>
<p><img src="/Users/macalesteritsloaner/Downloads/bookdown-demo-master/dataset/degreedistribution.png" /></p>
</div>
<div id="betti-number-inference" class="section level2" number="4.6">
<h2><span class="header-section-number">4.6</span> Betti Number Inference</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="persistence-homology.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="random.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/04-liegroupmorphism.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
