<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Persistence Homology | Geometry and topology statistical inference with applications.</title>
  <meta name="description" content="Capstone Project." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Persistence Homology | Geometry and topology statistical inference with applications." />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Capstone Project." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Persistence Homology | Geometry and topology statistical inference with applications." />
  
  <meta name="twitter:description" content="Capstone Project." />
  

<meta name="author" content="Emil Le" />


<meta name="date" content="2021-03-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="statistics-on-riemannian-manifold.html"/>
<link rel="next" href="application.html"/>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-3.5.1/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="libs/rglWebGL-binding-0.104.16/rglWebGL.js"></script>
<link href="libs/rglwidgetClass-0.104.16/rgl.css" rel="stylesheet" />
<script src="libs/rglwidgetClass-0.104.16/rglClass.min.js"></script>
<script src="libs/CanvasMatrix4-0.104.16/CanvasMatrix.min.js"></script>
<link href="libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<link href="libs/combineWidgetStyle-0.1/combineWidgets.css" rel="stylesheet" />
<script src="libs/combineWidgets-binding-0.10.1/combineWidgets.js"></script>
<script src="libs/rglPlayer-binding-0.104.16/rglPlayer.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="preface.html"><a href="preface.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#what-is-a-manifold"><i class="fa fa-check"></i><b>1.1</b> What is a Manifold?</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="statistics-on-riemannian-manifold.html"><a href="statistics-on-riemannian-manifold.html"><i class="fa fa-check"></i><b>2</b> Statistics on Riemannian Manifold</a>
<ul>
<li class="chapter" data-level="2.1" data-path="statistics-on-riemannian-manifold.html"><a href="statistics-on-riemannian-manifold.html#riemannian-manifold"><i class="fa fa-check"></i><b>2.1</b> Riemannian Manifold</a></li>
<li class="chapter" data-level="2.2" data-path="statistics-on-riemannian-manifold.html"><a href="statistics-on-riemannian-manifold.html#instrinsic-mean-and-variance"><i class="fa fa-check"></i><b>2.2</b> Instrinsic mean and variance:</a></li>
<li class="chapter" data-level="2.3" data-path="statistics-on-riemannian-manifold.html"><a href="statistics-on-riemannian-manifold.html#sample-fréchet-mean-and-variance"><i class="fa fa-check"></i><b>2.3</b> Sample Fréchet mean and variance:</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="persistence-homology.html"><a href="persistence-homology.html"><i class="fa fa-check"></i><b>3</b> Persistence Homology</a>
<ul>
<li class="chapter" data-level="3.1" data-path="persistence-homology.html"><a href="persistence-homology.html#vietorisrips-complex"><i class="fa fa-check"></i><b>3.1</b> Vietoris–Rips complex</a></li>
<li class="chapter" data-level="3.2" data-path="persistence-homology.html"><a href="persistence-homology.html#filtrations"><i class="fa fa-check"></i><b>3.2</b> Filtrations</a></li>
<li class="chapter" data-level="3.3" data-path="persistence-homology.html"><a href="persistence-homology.html#reeb-graph-inference"><i class="fa fa-check"></i><b>3.3</b> Reeb Graph inference</a></li>
<li class="chapter" data-level="3.4" data-path="persistence-homology.html"><a href="persistence-homology.html#confidence-set"><i class="fa fa-check"></i><b>3.4</b> Confidence Set</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="application.html"><a href="application.html"><i class="fa fa-check"></i><b>4</b> Application</a>
<ul>
<li class="chapter" data-level="4.1" data-path="application.html"><a href="application.html#gene-expression"><i class="fa fa-check"></i><b>4.1</b> Gene Expression</a></li>
<li class="chapter" data-level="4.2" data-path="application.html"><a href="application.html#generative-model-for-gene-co-expression-network"><i class="fa fa-check"></i><b>4.2</b> Generative model for gene co-expression network</a></li>
<li class="chapter" data-level="4.3" data-path="application.html"><a href="application.html#data-collection-and-pre-processing"><i class="fa fa-check"></i><b>4.3</b> Data Collection and Pre-processing</a></li>
<li class="chapter" data-level="4.4" data-path="application.html"><a href="application.html#network-visualization"><i class="fa fa-check"></i><b>4.4</b> Network Visualization</a></li>
<li class="chapter" data-level="4.5" data-path="application.html"><a href="application.html#heavy-tailed-distribution-analysis"><i class="fa fa-check"></i><b>4.5</b> Heavy tailed distribution analysis</a></li>
<li class="chapter" data-level="4.6" data-path="application.html"><a href="application.html#betti-number-inference"><i class="fa fa-check"></i><b>4.6</b> Betti Number Inference</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="random.html"><a href="random.html"><i class="fa fa-check"></i><b>5</b> random</a></li>
<li class="chapter" data-level="6" data-path="code-need-to-fix.html"><a href="code-need-to-fix.html"><i class="fa fa-check"></i><b>6</b> code need to fix</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Geometry and topology statistical inference with applications.</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="persistence-homology" class="section level1" number="3">
<h1><span class="header-section-number">Chapter 3</span> Persistence Homology</h1>
<div id="vietorisrips-complex" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Vietoris–Rips complex</h2>
<p>The n-simplex (simplicial complex) <span class="math inline">\(\sigma^n \subset R^{n+1}\)</span> is defined as:
<span class="math display">\[\sigma^n = \{(x_0,\dots,x_n)\in \mathbb{R}^{n+1}: x_i \geq 0 \text{ } \forall i; \sum_{i = 0}^n\}\]</span></p>
<p>The idea is we can deconstruct an object (such as a network) into more simple form (such as point and edges). Here, a 0-simplex is a point and the 1-simplex is a segment and so on.</p>
<p>Vietoris–Rips complex is the abstract simplicial complex whose k-simplices correspond to subsets of points in P with diameter at most <span class="math inline">\(2t\)</span>,
<span class="math display">\[\mathcal{R}(P,t) = \{ \sigma |\emptyset \neq \sigma \subset P, Diam(\sigma) \leq 2t\} \]</span>
## Persistent Diagram</p>
<p>At each intervals <span class="math inline">\((d_1,d_2)\)</span>, we record the homology at radius <span class="math inline">\(r_t\)</span> through time. At the end, we can have the Persistent diagram uniquely identified the point cloud. Persistent homology is robust against noise, where we can quantify our metric distance by the bottleneck distance:
<span class="math display">\[W_{\infty}(X,Y) = \inf_{\Gamma: X\to Y} \sup_{x \in X} \|x -\Gamma(x) \|_{\infty}\]</span>
We then can compare the method with other cluster algorithm such as principle component analysis, support vector machine or hierarchical clustering.</p>
</div>
<div id="filtrations" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Filtrations</h2>
<p>Point cloud <span class="math inline">\(\mathcal{X}\)</span> is simply the representation of data in space, then by replacing <span class="math inline">\(\mathcal{X}\)</span> with a nested interval Vietoris-Rips complexes. By taking the homology of each in the filtration of simplicial complexes, we can have obtain the persistent module as our features vectors for computation:
<span class="math display">\[H_i(\mathcal{X}_{r_0}) \to H_i(\mathcal{X}_{r_1}) \to H_i(\mathcal{X}_{r_2}) \to \dots\]</span>
The input of our problem is the gene expression matrix which can be represented as an object in <span class="math inline">\(\mathbb{R}^3\)</span> spaces: the expression value, the sample index and the gene name. The Mapper utilizes this property to preserve the local relationships of high dimensional expression data while clustering.</p>
</div>
<div id="reeb-graph-inference" class="section level2" number="3.3">
<h2><span class="header-section-number">3.3</span> Reeb Graph inference</h2>
<p>For constant threshold <span class="math inline">\(\epsilon = 10^{-6}\)</span>, let the metric space <span class="math inline">\((X,\partial_X)\)</span> where <span class="math inline">\(X\)</span> is the gene expression matrix and <span class="math inline">\(\partial_X: X\times X \to \mathbb{R}\)</span> defined by the Pearson’s correlation:
<span class="math display">\[\partial_X = \frac{\\Cov(X,X)}{\sigma_X^2}\]</span>
Let the filter function <span class="math inline">\(f: X \to \mathbb{R}^{n}\)</span> finds the first 2 nearest neighbors of each element in <span class="math inline">\(X\)</span> in the Euclidean norm <span class="math inline">\(\mathcal{L}^2\)</span>, then <span class="math inline">\(\mathcal{C} = \{U_{\alpha}\}\)</span> is a cover of the range of <span class="math inline">\(f\)</span> in <span class="math inline">\(\mathbb{R}\)</span>. Define the equivalence relation, <span class="math inline">\(\sim\)</span>, such that <span class="math inline">\(a \sim b\)</span> if <span class="math inline">\(a,b \in f^{-1}(U_{\alpha})\)</span> for arbitrary <span class="math inline">\(\alpha\)</span>. The Reeb space of X is the quotient of X by this equivalence relation. For appropriate choices of interval and percentages overlaps, we can have:</p>
</div>
<div id="confidence-set" class="section level2" number="3.4">
<h2><span class="header-section-number">3.4</span> Confidence Set</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="statistics-on-riemannian-manifold.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="application.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/03-homologyinference.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
